.gitlab-ci.yml

```diff
@@ -260,10 +231,13 @@ Smoke Tests (watchOS):
     - specific:true
   variables:
     XCODE: "15.2.0"
-    OS: "10.2"
   script:
-    - ./tools/runner-setup.sh --xcode "$XCODE" --watchOS --os "$OS" # temporary, waiting for AMI
+    - ./tools/runner-setup.sh --xcode "$XCODE" --iOS --tvOS --visionOS --watchOS
```

Comment:
💡 Current `macos:ventura` AMI includes all these platforms, but there is no harm in listing them here - the script will just skip what is already installed.